docs/processing/functional-connectivity.md

```diff
@@ -64,4 +65,13 @@ In the end, the data structure will look like this:
 │               ├── figures
 │               └── ses-15
 │                   └── func
-```
\ No newline at end of file
+```
+
+## QA/QC of denoised data
+
+- [ ] Run ... to generate the QC report for denoised data.
```

Comment:
Run ... ?

I guess we want to either replace the elipsis or open an issue and add a comment here to remark that this is TODO.